.yarn/releases/yarn-3.2.3.cjs

```diff
@@ -78,7 +78,7 @@ See https://chevrotain.io/docs/guide/resolving_lexer_errors.html#UNREACHABLE`;e.
 `),type:Ar.LexerDefinitionErrorType.LEXER_DEFINITION_CANNOT_CONTAIN_UNDEFINED})})}),i}Ze.performRuntimeChecks=hEe;function pEe(r,e,t){var i=[],n=!1,s=(0,Ne.compact)((0,Ne.flatten)((0,Ne.mapValues)(r.modes,function(l){return l}))),o=(0,Ne.reject)(s,function(l){return l[Ko]===Ar.Lexer.NA}),a=HH(t);return e&&(0,Ne.forEach)(o,function(l){var c=KH(l,a);if(c!==!1){var u=ZH(l,c),g={message:u,type:c.issue,tokenType:l};i.push(g)}else(0,Ne.has)(l,"LINE_BREAKS")?l.LINE_BREAKS===!0&&(n=!0):(0,mg.canMatchCharCode)(a,l.PATTERN)&&(n=!0)}),e&&!n&&i.push({message:`Warning: No LINE_BREAKS Found.
 	This Lexer has been defined to track line and column information,
 	But none of the Token Types can be identified as matching a line terminator.
-	See https://chevrotain.io/docs/guide/resolving_lexer_errors.html#LINE_BREAKS 
+	See https://chevrotain.io/docs/guide/resolving_lexer_errors.html#LINE_BREAKS
```

Comment:
Huh, what is going on here? This file is auto-generated. Maybe indicates that these changes should be undone, and this file added to the ESLint ignore.

Reply:
Hmm, yeah. We should probably ignore everything in `.yarn` in ESLint?

I've added it to the ignore list, and ran `yarn set version 3.2.3` to revert these changes.

Reply:
MetaMask/metamask-module-template#156